### ISSUE TITLE
Ignored [name] because: not enough data for decision.

### ISSUE THREAD

**Issue by abrkn** [crockpotveggies/tinderbox#83]
Always ignores everyone because not enough data for decision. Maybe I've been too picky :)

**Comment by twof** [comment]
+1

**Comment by jeraldjunkmail** [comment]
+1. It has not yet shown me a composite of "not like"'s. I think this is the root of the issue.  It is not using the not like decisions when swiping left or right.  I am not a programmer but want to keep this project alive.  How can I contribute?

**Comment by crockpotveggies** [comment]
Although I dropped support for this project, it appears I mistakenly introduced a bug with, I think, commit: https://github.com/crockpotveggies/tinderbox/commit/893085520df5c3577144f496584562b1e3fa1412

For whatever reason the thread that is supposed to save the user's choice either dies or experiences an error. Logs don't reveal anything. I suspect it might be related to MapDB. You can revert back to an earlier version of the code which may possibly fix the issue.

**Comment by jeraldjunkmail** [comment]
Come back!  This project needs a programmer! Checked out your bio and interests, and understand why you have moved on, but for me, this is an important project.  I certainly don't have time to swipe all day and send messages.  I would rather spend that time supporting this project, but can't code.  I like that the barrier to entry is high and there is no executable for windows.  If anyone reads this, I am willing to contribute time and organizational and administrative support for this project.

**Comment by crockpotveggies** [comment]
I didn't want to say anything because I didn't want to set promises I can't meet...I've been working on a better AI in my spare time. Dumping Eigenfaces in favor of something better. It hasn't been trivial by any means so to set expectations properly I dropped support. I will admit I'm surprised to see just how much interest this has gotten, and you're encouraging me to crack the whip.

Regarding these existing issues, let me see if I can do something over the next week and track down the bug. I've set a reminder to follow up on this thread next Tuesday.

**Comment by jeraldjunkmail** [comment]
Thanks.  I know this sounds weird, but this program (or one like it) is going to radically change the online dating world if it ever sees any measure of popularity.  It will end things as we know them or evolve into an arms race, like so many other new technologies.  For now  though, I like being the only guy out there with the nuke.  I won't take anything but code commits as "promises".  :)

I am not so much interested in programming and technical aspect as I am in having a stable, hard to set up elitist program that nobody else has.  If they are motivated to get it running (like me, took me a few hours to learn basic git stuff and sort through the documentation), they will.

**Comment by jeraldjunkmail** [comment]
That being said, if I had the aptitude for programming, I would be in there with you like a dirty shirt.  Love machine learning, the concepts, etc.  Watson and successors are going to change the world.  The work that guys like you do is brilliant.  I am en ex network admin/A+ guy.  I never got into coding.  I don't "get it".  That said I have administrative and managerial talents that could be useful on a project like this.  I just haven't ever been involved in software development.  Can't wait to see what you are up to!  Enough chatter, feel free to message me anytime.

**Comment by crockpotveggies** [comment]
I think the best way to leave this as stable as possible without sacrificing tangential hours is to revert back to a previous stable version. I'll branch this current version into a "work-in-progress".

@jeraldjunkmail going to send you a message directly since I have a couple questions regarding your most recent message.

Best,
Justin

**Comment by jeraldjunkmail** [comment]
Please feel free to message me anytime! I am a student at present, so despite the 75 hour work weeks, my schedule is flexible enough to contribute...somehow...

**Comment by crockpotveggies** [comment]
@jeraldjunkmail hey tried to email you and for some reason GitHub deflected my email to jeraldjunkmail@users.github.com. Do you think you could send me a message to crockpotveggies@users.github.com and I'll reply?

**Comment by jeraldjunkmail** [comment]
you can contact me directly at jeraldjunkmail@gmail.com. Still very interested in contributing!

**Comment by Rolilink** [comment]
what was the latest stable version? I am pretty sure the issue is that dislike eingenfaces model is not being generated actually I can't see that model

**Comment by crockpotveggies** [comment]
I believe if you rebase to this commit it will be semi-stable again: https://github.com/crockpotveggies/tinderbox/commit/4415abebec70f0adcd445184dd973f752908b368

I actually haven't even had a chance to rebase this repo since [Bernie](http://www.bernie.ai/) is on the way. I probably won't now that the Bernie A.I. is officially in progress.

**Comment by Rolilink** [comment]
Yeah, I signed up for Bernie it seems like an interesting project. If you need some help, I am a front end dev have some skills with node and rails too. Really interested in what you guys are doing. I am not a data scientist but machine learning and AI definitively always catch my attention. You can contact me at me@rolilink.com 

—Rolando Perez - Front End Developer

Email: me@rolilink.com

Tel: (+507) 6705-3289

**Comment by crockpotveggies** [comment]
@Rolilink hey thanks for signing up and would love to have you as an early user! I'll send you an email later today with some details on the project and where we're at right now.

Best,
Justin

**Comment by Rolilink** [comment]
Thanks @crockpotveggies  waiting for that message

**Comment by blackxored** [comment]
@crockpotveggies I'm also interested in Bernie, I understand you probably want to make this commercial and are not looking for a lot of help, if any, but just putting it out there that I'd be able to help on my spare time if you need. I'm a Sr. Ruby/JavaScript developer, and on the path of learning Scala too.

**Comment by jonbstrong** [comment]
@crockpotveggies I'm on the waiting list for Bernie and would be great to be an early user and help out where I can. You can email me at jbailliestrong [at] yahoo.co.uk

**Comment by crockpotveggies** [comment]
Hey guys thanks for signing up for the Bernie waitlist! Would love to have you both for alpha testing. We're going to begin Alpha in January and we're just a month away. Stay-tuned!

**Comment by KaiGutzeit** [comment]
I was not able to generate the Eigenfaces Images for the Dislike Model as well.

![image](https://cloud.githubusercontent.com/assets/11773466/14616956/a0bf5b6e-05ac-11e6-8556-acff8a11c890.png)

Often I also get the "Heap Space" error. Dont know if this is associated with the known issue:

![image](https://cloud.githubusercontent.com/assets/11773466/14617003/cd023e80-05ac-11e6-8b92-ba7d1b7a7164.png)

**Comment by crockpotveggies** [comment]
Hi everyone,

I'm following up from my previous comments on issues earlier this year. I promised to keep everyone in the loop on the successor to this prototype project, and it's now available. Bernie A.I. is now live in the App Store, and you're welcome to give it a try. A lot more effort has been put into making Bernie work, and it's going to be a lot more convenient.

https://itunes.apple.com/us/app/bernie-a.i./id1068738111

You're still welcome to keep using Tinderbox, I have no plans to remove the Github project. Please keep in mind it is no longer supported.

Warmly,
Justin
